src/vorta/views/source_tab.py

```diff
@@ -1,64 +1,135 @@
 from PyQt5 import uic
-from ..models import SourceFileModel, BackupProfileMixin
-from ..utils import get_asset, choose_file_dialog
-from PyQt5.QtWidgets import QApplication, QMessageBox
+from ..models import SourceFileModel, BackupProfileMixin, SettingsModel
+from ..utils import get_asset, choose_file_dialog, pretty_bytes, FilePathInfoAsync
+from PyQt5 import QtCore
+from PyQt5.QtCore import QFileInfo
+from PyQt5.QtWidgets import QApplication, QMessageBox, QTableWidgetItem, QHeaderView
 import os
 
 uifile = get_asset('UI/sourcetab.ui')
 SourceUI, SourceBase = uic.loadUiType(uifile)
 
 
 class SourceTab(SourceBase, SourceUI, BackupProfileMixin):
+    updateThreads = []
+    
     def __init__(self, parent=None):
         super().__init__(parent)
         self.setupUi(parent)
+        
+        headerTxt=["Path","Type","Size","Elements Count"]
```

Comment:
```suggestion
        headerTxt=["Path","Type","Size","File and Folder Count"]
```

Reply:
Maybe like this, to save some space in horizontal direction?
```suggestion
        headerTxt=["Path","Type","Size","Folder /\nFile Count"]
```

Reply:
It would look strange on wide screens. Qt table automatically wrap words I believe. https://doc.qt.io/qt-5/qtableview.html#wordWrap-prop